pygmt/figure.py

```diff
@@ -401,89 +401,98 @@ def savefig(  # noqa: PLR0912
         if show:
             launch_external_viewer(str(fname))
 
-    def show(self, dpi=300, width=500, method=None, waiting=0.5, **kwargs):
+    def show(
+        self,
+        method: Literal["external", "notebook", "none", None] = None,
+        dpi: int = 300,
+        width: int = 500,
+        waiting: float = 0.5,
+        **kwargs,
+    ):
         """
         Display a preview of the figure.
 
-        Inserts the preview in the Jupyter notebook output if available,
-        otherwise opens it in the default viewer for your operating system
-        (falls back to the default web browser).
+        Inserts the preview in the Jupyter notebook output if available, otherwise opens
+        it in the default viewer for your operating system (falls back to the default
+        web browser).
 
-        :func:`pygmt.set_display` can select the default display method
-        (``"notebook"``, ``"external"``, ``"none"`` or ``None``).
+        :func:`pygmt.set_display` can select the default display method (``"notebook"``,
+        ``"external"``, ``"none"`` or ``None``).
 
-        The ``method`` parameter can also override the default display method
-        for the current figure. Parameters ``dpi`` and ``width`` can be used
-        to control the resolution and dimension of the figure in the notebook.
+        The ``method`` parameter can also override the default display method for the
+        current figure. Parameters ``dpi`` and ``width`` can be used to control the
+        resolution and dimension of the figure in the notebook.
 
-        **Note**: The external viewer can be disabled by setting the
-        PYGMT_USE_EXTERNAL_DISPLAY environment variable to **false**.
-        This is useful when running unit tests and building the documentation
-        in consoles without a Graphical User Interface.
+        The external viewer can be disabled by setting environment variable
+        **PYGMT_USE_EXTERNAL_DISPLAY** to ``"false"``. This is useful when running tests
+        and building the documentation to avoid popping up windows.
```

Comment:
```suggestion
        The external viewer can be disabled by setting the environment variable
        **PYGMT_USE_EXTERNAL_DISPLAY** to ``"false"``. This is useful when running tests
        and building the documentation to avoid popping up windows.
```